UIKItTemplate/Modules/Notifications/View/NotificationViewCell.swift

```diff
@@ -0,0 +1,187 @@
+// NotificationViewCell.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+
+/// View to show product with price
+final class NotificationViewCell: UITableViewCell {
+    // MARK: - Constants
+
+    static let reuseID = String(describing: NotificationViewCell.self)
+    /// Constants for NotificationViewCell
+    private enum Constants {
+        /// Standard inset from left or right side
+        static let sideInset: CGFloat = 12
+        /// Inset per views
+        static let interItemInset: CGFloat = 7
+        /// Width and height of avatarImageView
+        static let imageSize: CGFloat = 40
+        /// Title for subscribe button
+        static let buttonTitle = "Подписаться"
+    }
+
+    // MARK: - Visual Components
+
+    private lazy var avatarImageView: UIImageView = {
+        let imageView = UIImageView()
+        imageView.layer.cornerRadius = Constants.imageSize / 2
+        imageView.clipsToBounds = true
+        imageView.contentMode = .scaleAspectFill
+        return imageView
+    }()
+
+    private lazy var descriptionLabel: UILabel = {
+        let label = UILabel()
+        label.numberOfLines = 0
+//        label.attributedText = makeDescriptionLabelText(no)
+        return label
+    }()
+
+    private lazy var postImageView: UIImageView = {
+        let imageView = UIImageView()
+        imageView.image = .nature4
+        return imageView
+    }()
+
+    private lazy var subscribeButton: UIButton = {
+        let button = UIButton(type: .custom)
+        button.backgroundColor = .blueApp
+        button.setTitleColor(.white, for: .normal)
+        button.titleLabel?.font = UIFont.makeVerdanaBold(10)
+        button.setTitle(Constants.buttonTitle, for: .normal)
+        button.layer.cornerRadius = 8
+        return button
+    }()
+
+    // MARK: - Public Properties
```

Comment:
в этом блоке снизу Public Method
перенеси в нужно место и поправь название марки
```suggestion
    // MARK: - Public Methods
```

Reply:
Done